### ISSUE TITLE
maintaining multiple clusters

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach-prod#62]
I'd like to do something like (omitting other stuff such as possibly passing `--state` to terraform as well)

```bash
terraform apply --var=cluster=test1 --var=num_instances=3
terraform apply --var=cluster=test2 --var=num_instances=3
```

which would give me two three-node clusters running in parallel. Maybe the use of `key_name` in the naming of various resources could be replaced by `cluster` (defaulting to `key_name`).

Not a priority or requirement at this point, but I'm curious if that's easy to implement.

**Comment by tamird** [comment]
This is stale (belongs in the main repo at a minimum). Please reopen there if this is still relevant.
